source/agora/common/Config.d

```diff
@@ -156,7 +156,7 @@ public Config parseConfigFile (CommandLine cmdln)
     Node root = Loader.fromFile(cmdln.config_path).load();
 
     if (auto node = "network" in root)
-        enforce(root["network"] == NodeID.sequence,
```

Comment:
I don't even know how the compiler allowed comparing a string to a ubyte here..

Reply:
https://github.com/dlang-community/D-YAML/blob/d10d2b2981c27c786a5fbfb25f7d431e4637ace4/source/dyaml/node.d#L422